ui/apps/dashboard/src/components/Billing/Plans/utils.tsx

```diff
@@ -0,0 +1,91 @@
+import { getPeriodAbbreviation } from '@inngest/components/utils/date';
+
+import type { BillingPlan } from '@/gql/graphql';
+
+export enum PlanNames {
+  Free = 'Free Tier',
+  Basic = 'Basic',
+  Pro = 'Pro',
+  Enterprise = 'Enterprise',
+}
+
+export function processPlan(plan: BillingPlan) {
+  const { name, amount, billingPeriod, features } = plan;
```

Comment:
The plan `features` field is deprecated in favor of the new entitlements stuff. [This PR](https://github.com/inngest/monorepo/pull/3721) contains a new `entitlements` field that will be good for that. I'm updating that PR to include `history`, `runCount`, and `stepCount` now